x-pack/plugins/ml/public/application/routing/routes/new_job/recognize.tsx

```diff
@@ -49,38 +48,89 @@ export const checkViewOrCreateRouteFactory = (): MlRoute => ({
 });
 
 const PageWrapper: FC<PageProps> = ({ location, deps }) => {
-  const { id, index, savedSearchId }: Record<string, any> = parse(location.search, { sort: false });
-  const { context, results } = useResolver(
-    index,
-    savedSearchId,
-    deps.config,
-    deps.dataViewsContract,
+  const { id } = parse(location.search, { sort: false });
 
-    {
-      ...basicResolvers(deps),
-      existingJobsAndGroups: mlJobService.getJobAndGroupIds,
-    }
-  );
+  const { context, results } = useRouteResolver('full', ['canGetJobs'], {
+    getMlNodeCount,
```

Comment:
Can we use basicResolvers() here?

Reply:
Updated in https://github.com/elastic/kibana/pull/159176/commits/018c48948231395246006974d3ccf25a061eef75